tests/scripts/run_demos.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+"""Run the Mbed TLS demo scripts.
+"""
+import glob
+import subprocess
+import sys
+
+def run_demo(demo):
+    """Run the specified demo script. Return True if it succeeds."""
+    returncode = subprocess.call([demo])
```

Comment:
Whishlist item: I'd love this script to have a `-q` option where stdout of the demos is suppressed.

Reply:
I don't really see the point, but ok.